dojo/forms.py

```diff
@@ -247,6 +247,11 @@ class ProductForm(forms.ModelForm):
                                        queryset=Product_Type.objects.none(),
                                        required=True)
 
+    sla_configuration = forms.ModelChoiceField(label='SLA Configuration',
+                                        queryset=SLA_Configuration.objects.all(),
+                                        required=True,
+                                        initial='Default')
+
```

Comment:
This a good place for this, but I fear product maintainers could swap an SLA conf to something a bit more forgiving (thinking the worst of people here)

Possible solution would be to not display this field unless user has owner role (similar to how jira options are toggled based on enablement)

Reply:
How would the value be set during creation? I'm still learning Django's idioms...